spec/polaris-management-service.yml

```diff
@@ -850,9 +850,92 @@ components:
         - $ref: "#/components/schemas/Catalog"
         - type: object
           properties:
-            remoteUrl:
-              type: string
-              description: URL to the remote catalog API
+            connectionConfigInfo:
```

Comment:
How updates happen? It looks like changing any config property in an `ExternalCatalog` requires re-submitting the whole object... Is that so?

Having client re-submit credentials on every config change is probably not ideal :thinking: WDYT?